src/openforms/conf/base.py

```diff
@@ -527,6 +537,9 @@
 # *should* have the same effect...
 CELERY_WORKER_PREFETCH_MULTIPLIER = 1
 
+# Amount of times to retry the celery task before stopping
+CELERY_MAX_RETRIES = config("CELERY_MAX_RETRIES", default=10)
```

Comment:
Please document this parameter in the docs

Reply:
Additionally, this makes it seem like this is an official Celery setting, which it isn't (at least I can't find it on https://docs.celeryproject.org/en/stable/userguide/configuration.html)

If it's a custom setting (which I'm 99% certain it is), please name it appropriately. I suggest using `SUBMISSION_REGISTRATION_MAX_RETRIES`